docs/data-model.md

```diff
@@ -4,47 +4,47 @@ weight = 5
 nav = [
     "Overview",
     "Index",
-    "Column",
-    "Row",
     "Field",
     "Time Quantum",
-    "Attribute",
-    "Shard",
 ]
 +++
 
 ## Data Model
 
 ### Overview
 
-The central component of Pilosa's data model is a boolean matrix. Each cell in the matrix is a single bit; if the bit is set, it indicates that a relationship exists between that particular row and column.
+The central component of Pilosa's data model is the index. An index can be thought of as a boolean matrix in which each cell in the matrix is a single bit; if the bit is set, it indicates that a relationship exists between that particular row and column.
 
 Rows and columns can represent anything (they could even represent the same set of things as in a [bigraph](https://en.wikipedia.org/wiki/Bigraph)). Pilosa can associate arbitrary key/value pairs (referred to as attributes) to rows and columns, but queries and storage are optimized around the core matrix.
 
-Pilosa lays out data first in rows, so queries which get all the set bits in one or many rows, or compute a combining operation—such as Intersect or Union—on multiple rows, are the fastest. Pilosa categorizes rows into different *fields* and quickly retrieves the top rows in a field sorted by the number of columns set in each row.
+Pilosa categorizes rows into different *fields* and makes queries which get all the set bits in one or many rows, or compute a combining operation — such as Intersect or Union — on multiple rows and possbily multiple fields. Pilosa can also quickly retrieve the top rows in a field sorted by the number of columns set in each row.
+
+<div class="note">
+    <font size="5"><b>Shard</b></font>
+    <p>Indexes are segmented into groups of columns called shards (previously known as slices). Each shard contains a fixed number of columns, which is the ShardWidth. ShardWidth is a constant that can only be modified at compile time, and before ingesting data. The default value is 2<sup>20</sup>.</p>
```

Comment:
Let's try to get away from using the word "column" which I think adds confusion for those familiar with relational databases. I think we can also remove "(previously known as slices)". 

"Indexes are segmented into shards which are 2<sup>20</sup> bits wide by default. Each shard contains data for all fields and all rows within each field, but just the portion of each row which is relevant to the shard. The width is controlled by a compile-time constant in the code called `ShardWidth`.